Client/Frontend/UserContent/UserScripts/Scripts_Dynamic/ScriptHandlers/Sandboxed/PlaylistScriptHandler.swift

```diff
@@ -233,16 +246,14 @@ extension PlaylistHelper: UIGestureRecognizerDelegate {
   }
 }
 
-extension PlaylistHelper {
+extension PlaylistScriptHandler {
   static func getCurrentTime(webView: WKWebView, nodeTag: String, completion: @escaping (Double) -> Void) {
     guard UUID(uuidString: nodeTag) != nil else {
       log.error("Unsanitized NodeTag.")
       return
     }
-
-    let token = UserScriptManager.securityTokenString
-
-    webView.evaluateSafeJavaScript(functionName: "window.__firefox__.mediaCurrentTimeFromTag_\(token)", args: [nodeTag], contentWorld: .page, asFunction: true) { value, error in
+    
+    webView.evaluateSafeJavaScript(functionName: "window.__firefox__.mediaCurrentTimeFromTag", args: [nodeTag, Self.scriptId], contentWorld: .page, asFunction: true) { value, error in
```

Comment:
I get multiple errors in this helper on youtube:
```
TypeError: window.__firefox__.mediaCurrentTimeFromTag is not a function. (In 'window.__firefox__.mediaCurrentTimeFromTag('0c1da85d-a67c-4024-874a-1274ee4ae396', '579CDEFB-5719-4A8B-83BE-3660852D3C94')', 'window.__firefox__.mediaCurrentTimeFromTag' is undefined)
TypeError: window.__firefox__.playlistLongPressed is not a function. (In 'window.__firefox__.playlistLongPressed(186.33332824707031, 153.66665649414062, '579CDEFB-5719-4A8B-83BE-3660852D3C94')', 'window.__firefox__.playlistLongPressed' is undefined)
```

Reply:
Fixed. Executed in the correct sandbox now (for all scripts).